src/views/Tree.vue

```diff
@@ -216,13 +221,26 @@ export default {
     }
   },
 
-  data: () => ({
-    expandAll: null,
-    tasksFilter: {
-      id: null,
-      states: null,
-    },
-  }),
+  props: { initialOptions },
+
+  setup (props, { emit }) {
+    /**
+     * The selection expand/collapse option.
+     * @type {import('vue').Ref<string>}
+     */
+    const expandAll = useInitialOptions('expandAll', { props, emit }, ref(null))
```

Comment:
I don't think it makes sense to store this, as this is not really a setting but a trigger. For example, a user could click collapse all, then expand a selection of nodes. If they navigated away then back, they would find all nodes collapsed again. We don't currently have a way to store the expanded/collapsed state of every node in the tree as a whole.

```suggestion
    /**
     * Setting this to an empty array causes all nodes to collapse.
     * Setting it to an array of node types causes those node types to expand.
     * @type {import('vue').Ref<string[]|null>}
     */
    const expandAll = ref(null)
```